apps/frontend/src/app/3_organisms/ZeroLocForm/components/FormContent.tsx

```diff
@@ -105,16 +109,18 @@ export const FormContent: FC<FormContentProps> = props => {
   const collateralTabs = useMemo(
     () => [
       {
-        value: AmountType.Add,
+        amountType: AmountType.Add,
         label: t(
           props.hasTrove
             ? translations.adjustCreditLine.actions.addCollateral
             : translations.adjustCreditLine.actions.collateral,
         ),
+        activeClassName: 'bg-gray-70 text-primary-20',
```

Comment:
```suggestion
        activeClassName: ACTIVE_CLASSNAME,
```
I think this and line 123 should also use `ACTIVE_CLASSNAME` since it's exact same style